### PR TITLE
Add documentation about json2kinto, kinto2xml and xml-verifier commands.

### PR DESCRIPTION
**Pull request by Natim**
- [x] Document installation #8
- [x] Document json2kinto importation command #18 
- [x] Document kinto2xml exportation command #19

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,23 +1,150 @@
...
+#########
+amo2kinto
+#########
 
-Generate a blocklist.xml file from Kinto collections.
+
+Installation
+============
+
+To install the release, just use::
```
Comment (leplatrem) [inline README.rst]: nit: «just use» is superflous, we know that the following is a command to use/run :)

File: README.rst [hunk 2]
```diff
@@ -1,23 +1,150 @@
...
+
+    pip install amo2kinto
+
+
+To install the development environment, just run::
```
Comment (leplatrem) [inline README.rst]: ditto

File: README.rst [hunk 3]
```diff
@@ -1,23 +1,150 @@
...
+
+System dependencies
+-------------------
+
+In order to build ``amo2kinto`` dependencies you may need the following libraries:
+
+- Building **lxml** will require: ``libxml2-dev libxslt-dev python-dev``
+- Building **cryptography** will require: ``build-essential libffi-dev libssl-dev python-dev``
+
+If you have got error messages telling you a header file is missing, please install them:
```
Comment (leplatrem) [inline README.rst]: «Otherwise you'll run into errors like:»

File: README.rst [hunk 4]
```diff
@@ -1,23 +1,150 @@
...
+========================================
+
+The `addons-server <https://github.com/mozilla/addons-server/>`_ is
+able to export its blocklists database in JSON.
+
+You can use the ``json2kinto`` script to load this database into a
+kinto server::
+
+    json2kinto --server http://localhost:8888/v1 \
+               --amo-server https://addons.mozilla.org/  \
```
Comment (leplatrem) [inline README.rst]: nit: Is this argument mandatory ? Are there several servers (like for QA?) ?

File: README.rst [hunk 5]
```diff
@@ -1,23 +1,150 @@
...
+               --schema-file schemas.json
```
Comment (leplatrem) [inline README.rst]: Better introduce this schemas.json file before mentionning it

File: README.rst (hunk 4 above)
Comment (Natim) [inline README.rst]: Yes there are [plenty of them](https://github.com/mozilla/addons-server/#addons-server):

- http://addons-dev.allizom.org/
- https://addons.allizom.org/
- https://addons.mozilla.org/

File: README.rst [hunk 6]
```diff
@@ -1,23 +1,150 @@
...
+
+The last version of the schema file can be find here:
+https://github.com/mozilla-services/amo-blocklist-ui/blob/master/amo-blocklist.json
```
Comment (leplatrem) [inline README.rst]: Maybe move above with something like:

First, obtain the last JSON schema:
```
curl https://github.com/mozilla-services/amo-blocklist-ui/blob/master/amo-blocklist.json > schemas.json
```

File: README.rst [hunk 7]
```diff
@@ -1,23 +1,150 @@
...
+
+
+Importing blocklists from AMO into Kinto
+========================================
+
+The `addons-server <https://github.com/mozilla/addons-server/>`_ is
+able to export its blocklists database in JSON.
+
+You can use the ``json2kinto`` script to load this database into a
+kinto server::
```
Comment (leplatrem) [inline README.rst]: nit: Kinto

File: README.rst [hunk 8]
```diff
@@ -1,23 +1,150 @@
...
+
+
+
+Generate a blocklist.xml file from Kinto collections
+====================================================
+
+If you want to export blocklists stored in Kinto in the AMO XML export
+format, you can use the ``kinto2xml`` script::
+
+    kinto2xml -s http://localhost:8888
```
Comment (leplatrem) [inline README.rst]: no `v1` prefix ?

File: README.rst [hunk 9]
```diff
@@ -1,23 +1,150 @@
...
+
+
+**kinto2xml** gives you the ability to configure what you want to
+export and in which bucket and collection are the data stored.
```
Comment (leplatrem) [inline README.rst]: and from which bucket and collection.

File: README.rst [hunk 10]
```diff
@@ -1,23 +1,150 @@
...
-      -h, --help            show this help message and exit
-      -s SERVER, --server SERVER
-                            The location of the remote server (with prefix)
-      -a AUTH, --auth AUTH  BasicAuth token:my-secret
-      -b BUCKET, --bucket BUCKET
-                            Bucket name.
-      -v, --verbose         Show all messages.
-      -q, --quiet           Show only critical errors.
-      -D, --debug           Show all messages, including debug messages.
-      -o OUT, --out OUT     Output file.
```
Comment (leplatrem) [inline README.rst]: Actually I'd prefer having the complete output of `--help`. It's easier to maintain and gives the whole command pattern (`[-s SERVER] [-a AUTH] [-b BUCKET] `)

File: README.rst [hunk 11]
```diff
@@ -1,23 +1,150 @@
...
+
+Once you've imported the data and exported them, you may want to
+validate that both Kinto and AMO export the exact same data in XML.
+
+You can use the ``xml-verifier`` command to validate that both files
+are rendered the same way::
+
+    xml-verifier \
+        https://blocklist.addons.mozilla.org/blocklist/3/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/46.0/ \
+        https://kinto-reader.dev.mozaws.net/v1/blocklist/3/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/46.0/
```
Comment (leplatrem) [inline README.rst]: What is this URL ? 
I suggest that you use the local files first, and then mention that it supports a remote URL of a Kinto server with the kinto-amo plugin install.

Would be nice to give the equivalent kinto2xml command along: 

```
APPID="{ec8030f7-c20a-464f-9b0e-13a3a9e97384}"
kinto2xml --app $APPID -o generated-blocklists.xml
xml-verifier https://blocklist.addons.mozilla.org/blocklist/3/$APPID/46.0/ generated-blocklists.xml
```

File: README.rst (hunk 4 above)
Comment (leplatrem) [inline README.rst]: :+1:

File: README.rst (hunk 11 above)
Comment (Natim) [inline README.rst]: Yes good point, the URL is the one from kinto-amo but it is out of scope here.

File: README.rst (hunk 4 above)
Comment (leplatrem) [inline README.rst]: BTW rename arg to `addons-server` ?

File: README.rst [hunk 12]
```diff
@@ -1,23 +1,182 @@
...
+The script is able to synchronize (add new blocked items and remove old ones).
+
+However if a blocked item already exists it won't be altered.
+
+**json2kinto** gives you the ability to configure what you want to
+load, with which user credentials and from which bucket and collection.
+
+.. code-block::
+
+usage: json2kinto [-h] [--addons-server ADDONS_SERVER] [-s SERVER] [-a AUTH]
```
Comment (leplatrem) [inline README.rst]: missing indentation ? 

capture d ecran de 2016-05-11 11-47-33

File: README.rst [hunk 13]
```diff
@@ -1,23 +1,182 @@
...
+
+The `addons-server <https://github.com/mozilla/addons-server/>`_ is
+able to export its blocklists database in JSON.
+
+The script will create the bucket/collection with a given schema.
+
+The last version of the schema file can be find here:
+https://github.com/mozilla-services/amo-blocklist-ui/blob/master/amo-blocklist.json
+
+You can use the ``make update-schemas`` command to grab it.
```
Comment (leplatrem) [inline README.rst]: grab, or «to fetch its last version» (as you want)

File: README.rst [hunk 14]
```diff
@@ -1,23 +1,182 @@
...
-                            The location of the remote server (with prefix)
-      -a AUTH, --auth AUTH  BasicAuth token:my-secret
-      -b BUCKET, --bucket BUCKET
-                            Bucket name.
-      -v, --verbose         Show all messages.
-      -q, --quiet           Show only critical errors.
-      -D, --debug           Show all messages, including debug messages.
-      -o OUT, --out OUT     Output file.
+    APPID="{ec8030f7-c20a-464f-9b0e-13a3a9e97384}"
+    kinto2xml --app $APPID -o generated-blocklists.xml
```
Comment (leplatrem) [inline README.rst]: don't you need to specify server buckets etc ?

**Comment by leplatrem** [comment]
r+ with some comments :)

File: README.rst (hunk 14 above)
Comment (Natim) [inline README.rst]: It depends of you setup, but default should work. (json2kinto and kinto2xml uses the same defaults.)

**Pull request merged**